src/EnergyPlus/WeatherManager.cc

```diff
@@ -828,7 +832,14 @@ namespace WeatherManager {
                                                    "WinterDesignDay",
                                                    "CustomDay1",
                                                    "CustomDay2"});
-
+        static Array1D_string const SkyTempModelNames(7,
+                                                      {"Clark and Allen",
+                                                       "Schedule Value",
+                                                       "DryBulb Difference Schedule Value",
+                                                       "Dewpoint Difference Schedule Value",
+                                                       "Brunt",
+                                                       "Idso",
+                                                       "Berdahl and Martin"});
```

Comment:
I would have preferred a vector here.

Reply:
@Myoldmopar like a std::vector<string>? I thought this should be a const array.